### ISSUE TITLE
Joystick: emergencyStop not disconnected in stopPolling

### ISSUE THREAD

**Issue by dayjaby** [mavlink/qgroundcontrol#10791]
With all the other signals in src/Joystick/Joystick.cc, there is a pattern: The signal is (disconnected/) connected in startPolling and disconnected in stopPolling. Only emergencyStop looks like an exception here. Is there a reason for this? Doesn't this have security implications?

**Comment by JMare** [comment]
Yes, good find, I don't think that is intentional, I guess when the emergencyStop feature was added it didn't get added to stopPolling.
Its not too critical, at least in master, because stoppolling almost never gets called in normal operation, so I'd be surprised to see any real world case where the joystick sends the signal to the wrong vehicle. Should be fixed for sure. You could make a PR?
